monkey/infection_monkey/exploit/powershell.py

```diff
@@ -182,15 +183,15 @@ def _execute_monkey_agent_on_victim(self):
                 f"Failed to execute the agent binary on the victim: {ex}"
             )
 
-    def _copy_monkey_binary_to_victim(self, monkey_path_on_victim):
+    def _copy_monkey_binary_to_victim(self, monkey_path_on_victim: Path):
 
         temp_monkey_binary_filepath = f"monkey_temp_bin_{get_random_file_suffix()}"
 
         self._create_local_agent_file(temp_monkey_binary_filepath)
 
         try:
             logger.info(f"Attempting to copy the monkey agent binary to {self.host.ip_addr}")
-            self._client.copy_file(temp_monkey_binary_filepath, monkey_path_on_victim)
+            self._client.copy_file(temp_monkey_binary_filepath, str(monkey_path_on_victim))
```

Comment:
Let's have `PowerShellClient` accept Paths and worry about the path -> str conversion within the powershell client.